containers/docker/pwpush/Dockerfile

```diff
@@ -0,0 +1,66 @@
+# pwpush-ephemeral
+FROM ruby:3.2-alpine AS build-env
+
+LABEL maintainer='pglombardo@hey.com'
+
+# Required packages
+RUN apk upgrade --no-cache \ 
+    && apk add --no-cache build-base git curl tzdata zlib-dev nodejs yarn libc6-compat sqlite-dev mariadb-dev libpq-dev
+
+ENV APP_ROOT=/opt/PasswordPusher PATH=${APP_ROOT}:${PATH} HOME=${APP_ROOT}
+
+RUN mkdir -p ${APP_ROOT}
+COPY ./ ${APP_ROOT}/
+
+WORKDIR ${APP_ROOT}
+
+RUN gem install bundler
+
+ENV RACK_ENV=production RAILS_ENV=production
+
+RUN bundle config set without 'development private test' \
+    && bundle config set deployment 'true' \
+    && bundle config set with 'sqlite mysql postgres' \
+    && bundle install \
+    && yarn install
+
+RUN bundle exec rails assets:precompile
+
+# Removing unneccesary files/directories
+RUN rm -rf node_modules tmp/cache vendor/assets spec \
+    && rm -rf vendor/bundle/ruby/*/cache/*.gem \
+    && find vendor/bundle/ruby/*/gems/ -name "*.c" -delete \
+    && find vendor/bundle/ruby/*/gems/ -name "*.o" -delete
+
+################## Build done ##################
+
+FROM ruby:3.2-alpine
+
+LABEL maintainer='pglombardo@hey.com'
+
+# install packages
+RUN apk upgrade --no-cache \
+    && apk add --no-cache tzdata bash nodejs libc6-compat sqlite-dev mariadb-connector-c libpq
+
+# Create a user and group to run the application
+ARG UID=1000
+ARG GID=1000
+
+RUN addgroup -g "${GID}" pwpusher \
+  && adduser -D -u "${UID}" -G pwpusher pwpusher
+
+ENV LC_CTYPE=UTF-8 LC_ALL=en_US.UTF-8
+ENV APP_ROOT=/opt/PasswordPusher PATH=${APP_ROOT}:${PATH} HOME=${APP_ROOT}
+WORKDIR ${APP_ROOT}
+ENV RACK_ENV=private RAILS_ENV=private
+
+RUN mkdir -p ${APP_ROOT} && chown -R pwpusher:pwpusher ${APP_ROOT}
```

Comment:
Here again with `WORKDIR`.

I'm not sure how the best practice would be of one needs to set permission on the folder.  I did just:
```dockerfile
RUN adduser -D pwpusher -u $UID -h "$APP_ROOT"

WORKDIR ${APP_ROOT}
COPY --from=build-stage --chown=pwpusher:pwpusher "${APP_ROOT}" ./
```

Reply:
WORKDIR and COPY instructions seem to be sufficent